app/lib/actions/katello/host/erratum/install.rb

```diff
@@ -9,10 +9,19 @@ def plan(host, errata_ids)
             Type! host, ::Host::Managed
 
             action_subject(host, :errata => errata_ids)
-            plan_action(Pulp::Consumer::ContentInstall,
-                        consumer_uuid: host.content_facet.uuid,
-                        type:          'erratum',
-                        args:          errata_ids)
+            if Setting['erratum_install_batch_size'] && Setting['erratum_install_batch_size'] > 0
```

Comment:
@akofink thanks for the information, I would definitely add this to my knowledge.

Though, I would have kept this as below:
if Setting['erratum_install_batch_size'].to_i > 0